scylla-persistence/src/main/java/com/netflix/conductor/scylla/config/ScyllaProperties.java

```diff
@@ -74,6 +74,25 @@ public class ScyllaProperties {
     @DurationUnit(ChronoUnit.SECONDS)
     private Duration eventExecutionPersistenceTtl = Duration.ZERO;
 
+    private int ttlShardIdCache = 10;
+    private int lengthShardIdCache = 1500000;
+
+    public int getTtlShardIdCache() {
+        return ttlShardIdCache;
+    }
+
+    public int getLengthShardIdCache()  {
+        return lengthShardIdCache;
+    }
+
+    public void setTtlShardIdCache(int ttlShardIdCache) {
```

Comment:
Why would we need to set / reset this ?

Reply:
just provided setter for that property, all properties have setter in that class